tests/utils.go

```diff
@@ -2949,6 +2953,7 @@ func WaitUntilVMIReadyWithContext(ctx context.Context, vmi *v1.VirtualMachineIns
 
 	// Fetch the new VirtualMachineInstance with updated status
 	virtClient, err := kubecli.GetKubevirtClient()
+	PanicOnError(err)
```

Comment:
instead of panic here. I think `Expect(err).ToNot(HaveOccurred())` is more appropriate

Reply:
updated, PTAL.